diceRoll.py

```diff
@@ -8,8 +8,16 @@ def dicegame(total_players,maximum_points):
 	for i in range(0,total_players):
 		points.append(0)
 	if total_players==1:
+		print("As you cannot play this game alone,I will play against you!")
```

Comment:
**Comments inclusion**

Since we have already introduced comments to explain the current state, inclusion of multiple comments seems to be redundant. Kindly revisit the flow to ensure if the additional comments are required.